web3.md

```diff
@@ -580,30 +580,75 @@ WEB3 JSON RPC
            ...
          </account>
          <messages>
-           (.Bag => <message>
-             <msgID> !TXID:Int </msgID>
-             <txNonce>    #if isString( #getJSON("nonce",    J) ) #then #parseHexWord(#getString("nonce",J))    #else ACCTNONCE  #fi </txNonce>
-             <to>         #if isString( #getJSON("to",       J) ) #then #parseHexWord(#getString("to",J))       #else 0          #fi </to>
-             <txGasLimit> #if isString( #getJSON("gas",      J) ) #then #parseHexWord(#getString("gas",J))      #else 90000      #fi </txGasLimit>
-             <txGasPrice> #if isString( #getJSON("gasPrice", J) ) #then #parseHexWord(#getString("gasPrice",J)) #else GASP       #fi </txGasPrice>
-             <value>      #if isString( #getJSON("value",    J) ) #then #parseHexWord(#getString("value",J))    #else 0          #fi </value>
-             <data>       #if isString( #getJSON("data",     J) ) #then #parseByteStack(#getString("data",J))   #else .ByteArray #fi </data>
-             ...
-           </message>)
-         ...
-         </messages>
-         <gasPrice> GASP </gasPrice>
-      requires #parseHexWord(#getString("from",J)) ==Int ACCTADDR
+            ( .Bag
+           => <message>
+                <msgID> !TXID </msgID>
+                <txGasPrice> 1         </txGasPrice>
+                <txNonce>    ACCTNONCE </txNonce>
+                <txGasLimit> 90000     </txGasLimit>
+                ...
+              </message>
+            )
+          ...
+          </messages>
+
+    syntax KItem ::= "loadTX" Int JSON
+ // ----------------------------------
+    rule <k> loadTX _ { .JSONList } => . ... </k>
+
+    rule <k> loadTX _ { ("from": _, REST) => REST } ... </k>
+
+    rule <k> loadTX TXID { ("to": TO_STRING, REST) => REST } ... </k>
+         <message>
+           <msgID> TXID </msgID>
+           <to> ( _ => #parseHexWord(TO_STRING) ) </to>
+           ...
+         </message>
+
+    rule <k> loadTX TXID { ("nonce": NONCE_STRING, REST) => REST } ... </k>
+         <message>
+           <msgID> TXID </msgID>
+           <txNonce> ( _ => #parseHexWord(NONCE_STRING) ) </txNonce>
+           ...
+         </message>
+
+    rule <k> loadTX TXID { ("gas": GAS_STRING, REST) => REST } ... </k>
+         <message>
+           <msgID> TXID </msgID>
+           <txGasLimit> ( _ => #parseHexWord(GAS_STRING) ) </txGasLimit>
+           ...
+         </message>
+
+    rule <k> loadTX TXID { ("gasPrice": GPRICE_STRING, REST) => REST } ... </k>
+         <message>
+           <msgID> TXID </msgID>
+           <txGasPrice> ( _ => #parseHexWord(GPRICE_STRING) ) </txGasPrice>
+           ...
+         </message>
+
+    rule <k> loadTX TXID { ("value": VALUE_STRING, REST) => REST } ... </k>
+         <message>
+           <msgID> TXID </msgID>
+           <value> ( _ => #parseHexWord(VALUE_STRING) ) </value>
+           ...
+         </message>
+
+    rule <k> loadTX TXID { ("data": DATA_STRING, REST) => REST } ... </k>
```

Comment:
same

Reply:
For this case I had to add the `ByteArray` type to the `JSON` sort in `data.md`.